happy_camper/app/scrapers/aa_scraper.rb

```diff
@@ -0,0 +1,50 @@
+class AAScraper
+
+  def initialize(base_url="http://progress.appacademy.io/employers")
+    @agent = Mechanize.new
+    @page = @agent.get(base_url)
+  end
+
+  def load_graduates
+    cohort_pages.each do |cp|
+      month, year, city = cohort_details(cp)
+      cp.search('.student').each do |s|
+        student_details = student_details(s)
+
+        Graduate.create({
+          full_name: student_details[:name],
+          picture_url: student_details[:picture_url],
+          month_attended: month,
+          year_attended: year,
+          bootcamp_location: city,
+          linkedin_url: student_details["LinkedIn"],
```

Comment:
unless you're planning to interpolate, single quotes are preferred. im nitpicking so you can ignore.